packages/strapi-connector-bookshelf/lib/queries.js

```diff
@@ -165,36 +160,21 @@ module.exports = function createQueryBuilder({ model, modelKey, strapi }) {
   }
 
   function search(params, populate) {
-    // Convert `params` object to filters compatible with Bookshelf.
-    const filters = modelUtils.convertParams(modelKey, params);
+    const filters = convertRestQueryParams(_.omit(params, '_q'));
 
     return model
-      .query(qb => {
-        buildSearchQuery(qb, model, params);
-
-        if (filters.sort) {
-          qb.orderBy(filters.sort.key, filters.sort.order);
-        }
-
-        if (filters.start) {
-          qb.offset(_.toNumber(filters.start));
-        }
-
-        if (filters.limit) {
-          qb.limit(_.toNumber(filters.limit));
-        }
-      })
-      .fetchAll({
-        withRelated: populate,
-      })
+      .query(buildSearchQuery({ model, params }))
```

Comment:
to avoid the sub query builder nesting in buildSearchQuery

What do you think of ? 

```js
module.query(qb => {
  qb.where(buildSearchQuery({ model, params }))
    .where(buildQuery({ model, filters )))
})
```

This makes it clearer we are doing an AND between both of those queries I think

If you are not a fan at least please avoid the setSearchColumn func as it is redefined everytime yo do a search